### DISCUSSION TITLE
Quasar: Instant server type change feedback

### DISCUSSION THREAD

**Discussion by Pranav-Bobde** [trpc/trpc#3597] [Q&A]
I've followed https://www.youtube.com/watch?v=GC_5p8bmnzI&t=912s and created a server and quasar-app (as a client).
And it's inferring correctly.

Problems:
1. Changes in the server type definitions, are not reflected on the client. (I'm able to get the updated changes by reloading my vs-code window).

2. Renaming (**Rename symbol**) a field not changing/reflecting on the client

Repo: https://github.com/Pranav-Bobde/demo-quasar-trpc

### Screenshots:

Renaming type definition on the server
![image](https://user-images.githubusercontent.com/66965591/211671614-7d0281a1-8570-4f14-be73-a3105c39d11a.png)

But no error reflected on the client
![image](https://user-images.githubusercontent.com/66965591/211672676-fda8adc3-4c6f-459f-a47f-8181af18aed8.png)

Have spent hours trying to get tRPC working with Quasar. 
Apparently just following the docs quick-start didn't work out for me. Couldn't get it to work properly work with v10.
Just found this one tutorial (with v9). Even in this, I don't really get much of the setup being done cause none of it's mentioned anywhere(docs) so yeah just getting to set it up and work has been a very tough ride (maybe because there is a lack of resources for support for quasar).
But just cause I love the possibilities of what it can do and help improve DX, I want to try it anyway.

Thanks.
Re

**Comment by juliusmarminge** [comment]
1. I'd suggest creating a proper monorepo with a single root lockfile and setup workspaces

2. V9 doesn't support "full stack refactoring" I.e rename symbol on backend won't rename on the frontend. This only works on v10
